### PR TITLE
Test prometheus endpoint existence.

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
This PR adds a functional test to see that a prometheus endpoint is created when importing and data is being generated.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by awels** [comment]
ci test please

File: pkg/image/qemu.go [hunk 1]
```diff
@@ -136,7 +136,16 @@ func (o *qemuOperations) Resize(image string, size resource.Quantity) error {
 }
 
 func (o *qemuOperations) Info(image string) (*ImgInfo, error) {
-	output, err := qemuExecFunction(qemuInfoLimits, nil, "qemu-img", "info", "--output=json", image)
```
Comment (mhenriks) [inline pkg/image/qemu.go]: Was the rate limited http server actually causing this to time out?  If so, cool, I guess.

File: tests/utils/common.go [hunk 2]
```diff
@@ -20,6 +20,8 @@ const (
 	HTTPAuthPort = 81
 	// HttpNoAuthPort provides a cdi-file-host service no-auth port for tests, requires AccessKeyValue and SecretKeyValue
 	HTTPNoAuthPort = 80
+	// HTTPRateLimit provides a cdi-file-host service rate limit port for tests, speed is limited to 25k/s to allow for testing slow connection behavior. No auth.
+	HTTPRateLimit = 82
```
[the remark this answers is not in the document]
Comment (awels) [reply tests/utils/common.go]: Yes should be consistent

File: pkg/image/qemu.go (hunk 1 above)
Comment (awels) [reply pkg/image/qemu.go]: Yeah, the default timeout of 5 seconds was not enough with the rate limited http server to get all the information qemu-img info wanted to obtain. I was surprised as well.

**Comment by awels** [comment]
ci test please

**Comment by awels** [comment]
ci test please

**Comment by awels** [comment]
ci test please

**Pull request merged**